### PR TITLE
Update phashion to current master

### PR DESCRIPTION
**Pull request by oteomoura**
[Asana](https://app.asana.com/0/187295284589485/1205054875862733/f)

### CODE REVIEW

**Review by davidpaniz — APPROVED** [review]

File: ext/phashion_ext/extconf.rb [hunk 1]
```diff
@@ -40,7 +41,7 @@
     system("cp -f libpHash.a libpHash_gem.a")
     system("cp -f libpHash.la libpHash_gem.la")
   end
-  $LIBS = " -lpthread -lpHash_gem -lstdc++ -ljpeg -lpng -lmvec"
+  $LIBS = " -lpthread -lpHash_gem -lstdc++ -ljpeg -lpng -lm"
```
Comment (gate8team) [inline ext/phashion_ext/extconf.rb]: I'm not 100% sure, but could we test if it works with svgs?
`lm` - this is the math lib
`lmvec` - this one for vectorized algebra

Very likely we need both (this is just a guess to prove or deny 😄 )

I also tried to google is there any difference between them and looks like there is and i'm not sure and can be wrong 100%, but let's test it with svgs before we deploy please? 🙏 

This one comes from ChatGPT (we still need to verify this)
`
In summary, lm is used for fitting linear regression models, while lmvec could potentially refer to a library or function that performs vectorized linear algebra operations.
They serve different purposes and are used in different contexts.
`

**Review by gate8team — CHANGES_REQUESTED** [review]

File: ext/phashion_ext/extconf.rb (hunk 1 above)
Comment (oteomoura) [reply ext/phashion_ext/extconf.rb]: @gate8team Sure! I'll fetch the latest master of `phashion` locally and try to calculate the phash for a SVG with it

Comment (gate8team) [reply ext/phashion_ext/extconf.rb]: Thanks 🙏

Comment (oteomoura) [reply ext/phashion_ext/extconf.rb]: @gate8team With a little delay, but just confirmed the current version of phashion in Artisan calculates the same phash value as the master of the original repo. I think we're good to proceed but you raised a valid point - in the least signal of trouble we add `-lmvec` back in.

image

**Review by gate8team — APPROVED** [review]
